zubhub_frontend/zubhub/src/components/materialsUsed/materialsUsed.jsx

```diff
@@ -0,0 +1,127 @@
+import React, {useState} from 'react';
+import AddMore from '../../components/addMore/addMore';
+import clsx from 'clsx';
+import ImageIcon from '@material-ui/icons/Image';
+import AddIcon from '@material-ui/icons/Add';
+import CustomButton from '../../components/button/Button';
+import Input from '../input/input'
+import {
+  Grid,
+  Box,
+  Container,
+  Card,
+  CardActionArea,
+  CardContent,
+  Chip,
+  Dialog,
+  Select,
+  MenuItem,
+  Typography,
+  CircularProgress,
+  OutlinedInput,
+  FormHelperText,
+  FormControl,
+  InputLabel,
+  ClickAwayListener,
+} from '@material-ui/core';
+import {handleImageFieldChange} from '../../views/create_activity/createActivityScripts';
+import UploadFile from '../../components/upload_file/uploadFile';
+
+function MaterialsUsed(props) {
+  const refs = {
+    imageInput: React.useRef(null),
+    image_count_el: React.useRef(null),
+  }
```

Comment:
The following lines are cleaner than creating refs object
```suggestion
  const imageInput = React.useRef(null);
  const image_count_el = React.useRef(null);
```